dangerfile.js

```diff
@@ -0,0 +1,71 @@
+import {message, danger, markdown} from "danger"
+
+const fs = require('fs').promises;
+const gzipSize = require('gzip-size');
+const git = require('simple-git/promise')(__dirname);
+
+// https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript
+function formatBytes(bytes, decimals = 2) {
+	if (bytes === 0) return '0 Bytes';
+
+	const k = 1024;
+	const dm = decimals < 0 ? 0 : decimals;
+	const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+	const i = Math.floor(Math.log(bytes) / Math.log(k));
+
+	return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+}
+
+const percDiff =  (first, second) =>
+	`${(100 * Math.abs( (first - second) / ( (first+second)/2 ) )).toFixed(2)}%`
+
+const run = async () => {
+	const minified = danger.git.modified_files.filter(file => file.includes('.min.js'))
+
+	if (minified.length === 0) {
+		return markdown(`# No JS Changes`);
```

Comment:
This is very misleading.

Also, if there is nothing to report, do have to make a message at all?

```suggestion
		markdown(`# No JS Changes`);
		return;
```

Reply:
We don't _have_ to, but it does make it explicit that there is no change (and that Danger ran successfully and found nothing worth reporting).

Reply:
Not sure I understand the "misleading" bit tho.

Reply:
> it does make it explicit that there is no change

That's a good point. Let's keep it then.

> Not sure I understand the "misleading" bit tho.

Yeah, I kinda changed the subject in the second sentence... With `return markdown(...)`, I expect that the returned value of `run` will be used which isn't the case. The return type of the `run` function is `void`. Please don't save a line by returning the result of `markdown(...)`.

Reply:
Fair enough. Fixed that.